techniques/flash/FLASH22.html

```diff
@@ -30,7 +30,7 @@ <h3>MovieClip used as a button</h3>
     						Button component, or create a new symbol with a type of "button". </p>
             </div>
          
-         <pre xml:space="preserve">import flash.accessibility. *
+         <pre>import flash.accessibility. *
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`